app/views/content/funding-and-support/scholarships-and-bursaries.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "Teacher training bursaries"
-heading: "Bursaries and scholarships"
+title: "Teacher training bursaries and scholarships"
+heading: "Teacher training bursaries and scholarships"
```

Comment:
url has scholarships and bursaries so is the other way round. Does this matter? Any SEO implications?